share/module_check_a_mundo.F

```diff
@@ -269,6 +269,73 @@ SUBROUTINE check_nml_consistency
          count_fatal_error = count_fatal_error + 1
       END IF
 
+!-----------------------------------------------------------------------
+! Check that the dimensions are correctly secified when sf_urban_physics = 2 and 3
+!-----------------------------------------------------------------------
+      DO i = 1, model_config_rec % max_dom
+         IF ( model_config_rec % sf_urban_physics(i) == bepscheme .OR. &
+              model_config_rec % sf_urban_physics(i) == bep_bemscheme )  THEN
+           IF ( model_config_rec % num_urban_ndm .NE. 2 ) THEN
```

Comment:
@smileMchen 
Ming,
Let's only trap if the user has DEFAULT settings.
Instead of 
```
IF ( model_config_rec % num_urban_ndm .NE. 2 ) THEN
```
Make it
```
IF ( model_config_rec % num_urban_ndm .EQ. 1 ) THEN
```
That would allow the urban developers to make these numbers whatever they want.

Reply:
@smileMchen @mgduda @barlage 
Ming,
Hold off on this suggestion.
Michael has an idea about checking the initial values.
We might need to get around some circular dependencies.